### DISCUSSION TITLE
[WP3940 Discussion]: Add connection to polyamine synthesis

### DISCUSSION THREAD

**Discussion by khanspers** [wikipathways/wikipathways-help#47] [General]
### Discussion

The following comments were communicated by email, from Kestutis Urba.

Suggested additions: 
- S-adenosyl-methionine conversion into decarboxylated s-adenosyl-methionine (and s-adenosylhomocysteine) to connect to polyamines synthesis.
- Add polyamine synthesis arm (?)

Evidence:
- https://www.sciencedirect.com/science/article/abs/pii/S0039914020311590 (graphical abstract)
- https://www.cureus.com/articles/148213-difluoromethylornithine-dfmo-and-neuroblastoma-a-review#!/ (fig 2)
- https://en.wikipedia.org/wiki/S-Adenosyl_methionine#Polyamine_biosynthesis

### Should this pathway be deleted?

- [ ] Pathway should be deleted

**Comment by KestUrba** [comment]
This  problem of SAM-polyamine connections was solved successfully by You and  Danidis yesterday. Thanks both. But there exist very important connections among mTor1 (-the great regulator of cell fate) and polyamines too: ref. **Gomes AP, Schild T, Blenis J. Adding Polyamine Metabolism to the mTORC1 Toolkit in Cell Growth and Cancer. Dev Cell. 2017 Jul 24;42(2):112-114. doi: 10.1016/j.devcel.2017.07.004. PMID: 28742999; PMCID: PMC5705022.**
Secondly, the so called imperator of genes c-myc regulates polyamines synthesis: ref. **Bachmann AS, Geerts D. Polyamine synthesis as a target of MYC oncogenes. J Biol Chem. 2018 Nov 30;293(48):18757-18769. doi: 10.1074/jbc.TM118.003336. Epub 2018 Nov 7. PMID: 30404920; PMCID: PMC6290138.**  
By the way, polyamines are universal regulators of transcription in almost all human  cells.  These connections were discussed privately by prof. Frank McCormic in February and called to be important. He is a world recognised oncologist investigating   molecular mechanisms of mutated K-ras in cancer cells. How these two mentioned connections, which seemed to be of great importance not only for me,  are represented in WikiPath? I'm afraid - ZERO. Because of a huge amount of information in the "ocean" of more than 5 mln. scientific articles and  fragmented way of research and corresponding mentality: analysis overcomes synthesis. This situation was named by the great philosopher Pierre Teilhard de Chardin to be **The curse of science** in ...1955.  The complexity of pathways is so great, that only Artificial Intelligence programs could find the seeds and roots and trunk having  branches and leaves in one drawing of carcinogenesis path in the near future, I hope. Can we  outdistance AI now? We must try because people are suffering from cancer and dying.  Regulation of polyamines   by mTor1, c-myc  seems to me to be the very significant not only for representation in WikiPath. Unfortunately, my two days' experience studying WikiPatways doesn't allow me to solve this immediately.

**Comment by khanspers** [comment]
Thanks for the suggestions. It seems to me that the connections between mTor-1 / c-myc and polyamines in cancer should be described in one or more dedicated pathways, and not part of the One-carbon metabolism pathway. 

Another resource closely related to WikiPathways is PFOCR, which is a project focused on extracting gene mentions from published pathway figures. We maintain a separate website for this project where you can search by keyword, gene, PMCID etc to find pathway figures from the past 25 years.  https://pfocr.wikipathways.org/
This database can be used to find pathways of interest to convert to WikiPathways models; in fact there is a "download gpml" option to the right under each pathway figure which will give you a "starter" pathway in gpml format with all genes found on the pathway.

For example, a search for "myc AND polyamine" yields several pathway figures, including one from the Bachman paper you mentioned (PMC6290138):
https://pfocr.wikipathways.org/search.html?query=myc%20polyamine

Im going to close this issue since the original issue was solved.
